gnosis/eth/clients/etherscan_client.py

```diff
@@ -44,7 +44,8 @@ class EtherscanClient:
         EthereumNetwork.CRONOS_TESTNET: "https://testnet.cronoscan.com",
         EthereumNetwork.CELO_MAINNET: "https://celoscan.io",
         EthereumNetwork.BASE_GOERLI_TESTNET: "https://goerli.basescan.org",
-        EthereumNetwork.NEON_EVM_DEVNET: "https://neonscan.org",
+        EthereumNetwork.NEON_EVM_DEVNET: "https://devnet.neonscan.org",
```

Comment:
devnet url explorer does not work. It is a temporary issue?

Reply:
It works again

Reply:
Looks like it was a temporary issue